### ISSUE TITLE
[They Vote For You/production] ArgumentError: Nil location provided. Can't build URI.

### ISSUE THREAD

**Issue by mlandauer** [openaustralia/publicwhip#1370]
## Backtrace

    line 8 of [PROJECT_ROOT]/app/views/card/policy_with_member_card.html.haml: _app_views_card_policy_with_member_card_html_haml___244850121232895712_2354220
    line 69 of [PROJECT_ROOT]/app/controllers/members_controller.rb: policy

[View full backtrace and more info at honeybadger.io](https://app.honeybadger.io/projects/37542/faults/84428300)

**Comment by mlandauer** [comment]
This is happening in production

**Comment by mlandauer** [comment]
@Taz17 @tseyiokorodudu could you please have a look at this bug please? It's happening on certain cards

**Comment by mlandauer** [comment]
This is one of the cards it's failing on https://theyvoteforyou.org.au/people/senate/tasmania/steve_martin/policies/15?card=true

**Comment by Taz17** [comment]
Looking at the backtrace on honeybager it seems that `image_tag` in https://github.com/openaustralia/publicwhip/blob/bd79993e8f0b7990649ef122218efd9fc55f0bf2/app/views/card/policy_with_member_card.html.haml#L8

Is receiving a string (or no string at all) which cannot be parsed to a URI as the exception message says `Nil location provided. Can't build URI.`

This error is not occurring with all other members. Which makes me think that not all members have a `extra_large_image_url`. Additionally, in pages like https://theyvoteforyou.org.au/people/senate/tasmania/steve_martin/policies/15 we use `large_image_url`

https://github.com/openaustralia/publicwhip/blob/bd79993e8f0b7990649ef122218efd9fc55f0bf2/app/views/policies/show_with_member.html.haml#L33

@mlandauer is there a way we can check the empty entries in the prod DB?

**Comment by Taz17** [comment]
Additionally, in this commit https://github.com/openaustralia/publicwhip/commit/03c3c3719f117fb2e0a46b2a140d69b38e2ecf35 we started using `extra_large_image_url` over `large_image_url`.

Looking at the git blame @mlandauer made this change. Why?

**Comment by tseyiokorodudu** [comment]
@Taz17  The `extra_large_image_url` was added so we didn't have to hardcode the image size like we did when we used `large_image_url`.  In other words, `extra_large_image_url` is the exact image size we need for the cards.

Also I agree with @Taz17 that the reason behind the error is that not all members have a `extra_large_image_url`. Had the same problem locally and @mlandauer you suggested I updated the member images by running the member load rake task (https://oaf.slack.com/archives/C4BLHGGC8/p1646007971924879). Could that be what needs to be done on the production end of things?
